requirements.txt

```diff
@@ -1,11 +1,11 @@
 # Package dependencies
-carbon-black-cloud-sdk >= 1.5.0
+carbon-black-cloud-sdk >= 1.5.4
```

Comment:
Needs updated to 1.5.5 and the same change made it setup.py

Reply:
Added.